scripts/release/sign-and-notarize-macos.sh

```
@@ -43,43 +43,76 @@ echo "[sign] Signing identity imported into $KEYCHAIN"
+# ── Nested Frameworks/ (CEF + Helper apps) ──────────────────────────────────
+# Must be signed from the inside out, before the outer .app bundle.
+if [ -d "$APP_PATH/Contents/Frameworks" ]; then
+  # 1. Sign loose dylibs / binaries inside any *.framework
+  while IFS= read -r -d '' fw; do
+    echo "[sign]   Scanning framework: $(basename "$fw")"
+    while IFS= read -r -d '' item; do
+      # Skip symlinks; codesign will sign the real file via Versions/Current
+      [ -L "$item" ] && continue
+      case "$item" in
+        *.dylib|*.so)
+          echo "[sign]     Signing lib: ${item#$APP_PATH/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)